src/js/views/patients/actions/actions_views.js

```diff
@@ -90,14 +90,7 @@ const OwnerComponent = Droplist.extend({
     if (this.getOption('isCompact')) {
       return {
         className: 'actions__owner button-secondary--compact w-100',
-        template: hbs`{{far "user-circle"}}{{ short }}{{ first_name }} {{ lastInitial }}`,
-        templateContext() {
-          return {
-            lastInitial() {
-              return this.last_name && `${ this.last_name[0] }.`;
-            },
-          };
-        },
+        template: hbs`{{far "user-circle"}}{{ short }}{{ name }}`,
```

Comment:
roles have names. so this looks like an issue to me.  either we'll need to split the template here or make a templateContext helper that returns the appropriate result based on the model type